### PR TITLE
Rewriting the firewalls doc

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

File: docs/kb/supporting-your-customers/firewalls.md [hunk 1]
```diff
@@ -1,22 +1,93 @@
...
+title = "Firewalls"
 weight = "999999"
 categories = [ "Knowledgebase", "Supporting Your Customers" ]
 +++
 
-## Replicated UI
+Often, customers will need to have a complete list of expected internal and outbound network traffic so they can open ports in firewalls and whitelist hosts and IP addresses for outbound connectivity. This document provides the list of all known connections that Replicated requires to run. Any external services required are not listed here.
 
-To utilize the Replicated management console you are required to allow inbound/outbound traffic on TCP port `:8800` of the appropriate network interface (`eth0`, `eth1`, etc.) on the host machine where Replicated is installed. This port only needs to be open to the subnet with which an IT administrator would be accessing the console to configure your application.
+Note: Airgap installations can run completely offline, and all tasks can be performed with not outbound internet access. Additionally, no installations of Replicated ever require inbound access.
```
Comment (stephenlawrence) [inline docs/kb/supporting-your-customers/firewalls.md:11]: with not -> without

File: docs/kb/supporting-your-customers/firewalls.md [hunk 2]
```diff
@@ -1,22 +1,93 @@
...
 
-## Replicated Services Communication
+Depending on the current activity, the needs can be different. This document is broken into the task that the customer is attempting to perform, and then broken down by the type of installation they are running.
```
Comment (stephenlawrence) [inline docs/kb/supporting-your-customers/firewalls.md:13]: the task -> the tasks

File: docs/kb/supporting-your-customers/firewalls.md [hunk 3]
```diff
@@ -1,22 +1,93 @@
...
+To install updates, some external connections are required. All connetions are initiated from inside the network, and can vary depending on the installation method and the application update.
 
-Make sure to include these requirements in any 
-[customer facing installation instructions](https://support.replicated.com/hc/en-us/articles/216652467) you create.
+| Host | Online Installation | Airgap Installation | Description |
+|---|---|---|---|
+
+
+## Ongoing Access
+When the application is up and running, and not being updated, the requirements for outbound internet access are greatly reduced. It's possible to even run a server completely disconnected from the Internet, and onlty connect when you want to check for updates.
```
Comment (stephenlawrence) [inline docs/kb/supporting-your-customers/firewalls.md:32]: onlty -> only

File: docs/kb/supporting-your-customers/firewalls.md [hunk 4]
```diff
@@ -1,22 +1,93 @@
...
+
+Once the application is installed, your customer can continue to run it, and start and start the application without any outbound access.
```
Comment (stephenlawrence) [inline docs/kb/supporting-your-customers/firewalls.md:34]: start and -> stop and

File: docs/kb/supporting-your-customers/firewalls.md [hunk 5]
```diff
@@ -1,22 +1,93 @@
...
+
+In order to perform basic maintence, some outbound access is required, as documented in the table below:
+
+| Task | Host | Online Installation | Airgap Installation | Description |
+|---|---|---|---|---|
+| Check for updates | api.replicated.com (port 443) | <i class="fa fa-check" /> Required | <i class="fa fa-times" /> Not Required | This endpoint is the only endpoint required to check for application updated. |
```
Comment (stephenlawrence) [inline docs/kb/supporting-your-customers/firewalls.md:40]: updated -> updates

File: docs/kb/supporting-your-customers/firewalls.md [hunk 6]
```diff
@@ -1,22 +1,93 @@
...
+* IP Addresses
+  * [Replicated Cloud IPs](https://github.com/replicatedhq/ips/blob/master/ip_addresses.json)
+  * Quay Cloud IPs
+* Hostnames
+  * get.replicated.com
+  * Quay Hostnames
+
+### Application Installation or Upgrade
+
+The application installation start with using the Replicated web console and finishes when your images are downloaded. To use the Replicated management console you are required to allow inbound/outbound traffic on TCP port `:8800` to the subnet with which an IT administrator would be accessing the console to configure your application.
```
Comment (stephenlawrence) [inline docs/kb/supporting-your-customers/firewalls.md:65]: start -> starts

**Review by stephenlawrence — CHANGES_REQUESTED** [review]

**Comment by mhald** [comment]
Closing, merged my updated copy.

**Pull request closed without merging**